src/helpers/runtime_error/index.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import { Errors } from 'Constants';
+const passableArgs:string = 'passableArgs';
+
+function errorBuilder(functionName, errorMessage) {
+    return `[Passable]: Failed to execute '${functionName}': ${errorMessage}`;
```

Comment:
In regards to my comment about the error naming convention (`src/constants/errors/index.js`) - Adding the error number here would make our lives easier and would be more constructive, even more so when we'll have numerous error messages.